packages/asset-swapper/src/quote_consumers/forwarder_swap_quote_consumer.ts

```diff
@@ -129,23 +131,21 @@ export class ForwarderSwapQuoteConsumer implements SwapQuoteConsumerBase<Forward
             params = {
                 orders,
                 signatures,
-                feeOrders,
-                feeSignatures,
-                feePercentage,
+                feePercentage: formattedFeePercentage,
                 feeRecipient: normalizedFeeRecipientAddress,
                 type: MarketOperation.Sell,
             };
             methodName = 'marketSellOrdersWithEth';
         }
-        const methodAbi = utils.getMethodAbiFromContractAbi(
-            this._contractWrappers.forwarder.abi,
-            methodName,
-        ) as MethodAbi;
+        const methodAbi = utils.getMethodAbiFromContractAbi(this._forwarder.abi, methodName) as MethodAbi;
 
+        const ethAmountWithFees = affiliateFeeUtils
+            .getTotalEthAmountWithAffiliateFee(worstCaseQuoteInfo, feePercentage)
+            .plus(worstCaseQuoteInfo.protocolFeeInEthAmount);
```

Comment:
Not sure if this is what we want, but here I just add protocol fees to the total payable amount.

Reply:
`getTotalEthAmountWithAffiliateFee` will consider protocol fees since affiliateFee is dependent on protocolFee (as of current forwarder design).

Reply:
ahh, yeah, that makes sense